docs/README-AWS.md

```diff
@@ -1,92 +1,132 @@
-  * Go to "Configuration" tab and select "Environment Variables"
-  * You can additionally add the following environment variables to enable Elastic APM instrumentation for your deployment of Elastic Serverless Forwarder
-    * | Key                       | Value  |
+There are several deployment methods available:
+
+### Deploy using AWS Console
+1. Login to AWS console and navigate to the Lambda service
+1. Click *Create function*
+1. Select *Browse serverless app repository*
+1. Select *Public applications*
+1. Search for *elastic-serverless-forwarder* and select it from results
+1. Complete the *Application settings* as follows:
+  * `ElasticServerlessForwarderS3ConfigFile` with the value of the S3 url in the format "s3://bucket-name/config-file-name" pointing to the configuration file for your deployment of Elastic Serverless Forwarder (see below), this will populate the `S3_CONFIG_FILE` environment variable of the deployed Lambda.
+  * `ElasticServerlessForwarderSSMSecrets` with a comma delimited list of AWS SSM Secrets ARNs referenced in the config yaml file (if any).
+  * `ElasticServerlessForwarderKMSKeys` with a comma delimited list of AWS KMS Keys ARNs to be used for decrypting AWS SSM Secrets referenced in the config yaml file (if any).
+  * `ElasticServerlessForwarderSQSEvents` with a comma delimited list of Direct SQS queues ARNs to set as event triggers for the Lambda (if any).
+  * `ElasticServerlessForwarderS3SQSEvents` with a comma delimited list of S3 SQS Event Notifications ARNs to set as event triggers for the Lambda (if any).
+  * `ElasticServerlessForwarderKinesisEvents` with a comma delimited list of Kinesis Data Stream ARNs to set as event triggers for the Lambda (if any).
+  * `ElasticServerlessForwarderCloudWatchLogsEvents` with a comma delimited list of Cloudwatch Logs Log Groups ARNs to set subscription filters on the Lambda for (if any).
+  * `ElasticServerlessForwarderS3Buckets` with a comma delimited list of S3 buckets ARNs that are the sources of the S3 SQS Event Notifications (if any).
+1. Click *Deploy* once your settings have been added
+1. On the Applications page for *serverlessrepo-elastic-serverless-forwarder*, click *Deployments*
+1. Refresh the *Deployment history* until status updates to `Create complete`
+1. (Optional) To enable Elastic APM instrumentation for your new deployment:
+  * Go to *Lambda > Functions* within AWS console, find and select the function with *serverlessrepo-elastic-se-ElasticServerlessForward-*
+  * Go to *Configuration* tab and select *Environment Variables*
+  * Add the following environment variables:
+
+```   | Key                       | Value  |
       |---------------------------|--------|
       |`ELASTIC_APM_ACTIVE`       | `true` |
       |`ELASTIC_APM_SECRET_TOKEN` | token  |
       |`ELASTIC_APM_SERVER_URL`	  | url    |
+```
```

Comment:
Did you intend for this to be a code block? 

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/10479155/176270987-9dd624d6-715e-48ca-a7fe-cb475d82557c.png">

```suggestion
      | Key                       | Value  |
      |---------------------------|--------|
      |`ELASTIC_APM_ACTIVE`       | `true` |
      |`ELASTIC_APM_SECRET_TOKEN` | token  |
      |`ELASTIC_APM_SERVER_URL`	  | url    |
```

☝️ I think this might be more appropriate:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/10479155/176271119-23c2befe-8a8a-4611-a3c0-910e460472e4.png">